hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/cache/TestFederationCache.java

```diff
@@ -63,15 +60,16 @@ public static Collection<Class[]> getParameters() {
   private FederationStateStoreTestUtil stateStoreTestUtil;
   private FederationStateStoreFacade facade;
 
-  public TestFederationCache(Class cacheClassName) {
+  public void initTestFederationCache(Class cacheClassName)
```

Comment:
Can this be made `private`?

Reply:
Thank you for reviewing this PR! I will work on improving it.